studio/util.py

```diff
@@ -275,6 +273,20 @@ def upload_file(url, local_path, logger=None):
         logger.debug('File upload done in {} s'
                      .format(time.time() - tic))
 
+def _looks_like_url(name):
+    """
+    Function tries to determine if input argument
+    looks like URL and not like S3 bucket name.
+    :param name - input name
+    :return: True, if name looks like URL;
+             False otherwise.
+    """
+    if name.endswith('.com'):
+        return True
+    if name.find(':') >= 0:
+        # Assume it is port number
+        return True
+    return False
```

Comment:
That's another bug fix: URLs are not only the ones ending with ".com",
but also, say, those with ":" in them (for port number).
That makes it work for Minio endpoints.

Reply:
Thanks the 381 fix